Tests/CarthageKitTests/DependencySpec.swift

```diff
@@ -84,6 +84,17 @@ class DependencySpec: QuickSpec {
 
 					expect(dependency.name) == "whatisthisurleven"
 				}
+
+                it("should be the directory name if the given URL is relative local path") {
```

Comment:
Same comment here about indentation.

Reply:
Fixed in https://github.com/Carthage/Carthage/pull/2907/commits/e561595a6fdfe2bc47ca79866104c396575d8183.